README.md

```diff
@@ -38,6 +38,10 @@
 
 Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting vulnerabilities in third-party dependencies, and enforcing code standards. [Get started →.](#getting-started)
 
+<img src="images/semgrep-main-image.jpg" /> 
+
+<details>
+  <summary style="font-size:16px;">Semgrep Ecosystem</summary>
```

Comment:
For now, it's hidden under the details toggle and so it can remain there.

Reply:
We added them to the README because we thought it is critical to include, and I don’t think we have updated information to show otherwise, so we should leave it in there. I think having it under the toggle dropdown might be ok, but I ask myself why it should be default hidden given its placement in the order of the README? 

Assuming many readers don't click on the toggle dropdown (it’s easy to miss), then there is much less context shown to the reader about what is Semgrep aside from the single opening sentence.

Reply:
discussed this with @pabloest and have moved it to after the Getting started section to keep the focus getting started.